src/proc.cpp

```diff
@@ -863,29 +863,31 @@ int Procinfo::readproc()
     {
         // rchar = ... not file maybe sockread
         //
-        mini_sscanf(buf, "read_bytes:%d", &io_read);   // io_read in Byte
-        mini_sscanf(buf, "write_bytes:%d", &io_write); // io_write in Byte
-        io_read /= 1024;
-        io_write /= 1024;
+        unsigned long io_r, io_w;
+        mini_sscanf(buf, "read_bytes:%lu", &io_r);  // io_r in Byte
+        mini_sscanf(buf, "write_bytes:%lu", &io_w); // io_w in Byte
+        io_read = io_r / 1024;
+        io_write = io_w / 1024;
 
         // if(io_read_prev!=0)
         {
             if (io_read_prev == 0)
-                io_read_prev = io_read;
+                io_read_prev = io_r;
             if (io_write_prev == 0)
-                io_write_prev = io_write;
+                io_write_prev = io_w;
 
-            // NOTE: Kbps right????
-            io_read_KBps = (io_read - io_read_prev) /
+            // NOTE: Kbps = byte / (msec / 1000) / 1024    // accurate
+            //       Kbps = (byte / 1024) / (msec / 1024)  // not accurate
+            io_read_KBps = (io_r - io_read_prev) /
                            proc->update_msec; // not accurate....
```

Comment:
Why didn't you make it more accurate? I mean something like:

```c++
io_read_KBps = ((1000 * (io_r - io_read_prev)) / (proc->update_msec)) / 1024;
```

Reply:
> Why didn't you make it more accurate? I mean something like:

I imagined what the original developer was thinking.

In his source code, there was `// not accurate....`.
He prioritized "fast processing speed" rather than "accurate value".
It was the code for non-powerful machines.
Multiplication and division are huge CPU load.
He thought what kind of executable code the C++ compiler output, and he coded it, i think.

I respected this source code and I left this unchanged.
So I used `io_r` and `io_w` as a different approach.

I was too much thinking about it.
I am sorry to have used your precious time.